app/controllers/application_controller.rb

```diff
@@ -2,8 +2,33 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  
+  VALUES = [
+           1001,
+           2020,
+           3333,
+         ]
 
-  def  aaa
-      1 + 1
+  def  aaa (a, b   , c,d,e,f    ,    g)
+    if a = 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.